src/components/Common/InternalError/InternalError.module.scss

```
@@ -0,0 +1,27 @@
+h1 {
+  font-size: toRem(21) !important;
+}
```

I think if you change this to

```suggestion
.internalErrorCardTitle {
  font-size: toRem(21);
}
```

it should work (unless this is what you already tried)

You're already applying the `internalErrorCardTitle` class via className so i would hope that the specificity of the css module class would beat out the h1 element styles

---

I tried this before and still get the same thing :(